### PR TITLE
Add osxfuse beta version v3.0.9

### PR DESCRIPTION
**Pull request by zchee**

### CODE REVIEW

File: Casks/osxfuse-beta.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask 'osxfuse-beta' do
+  version '3.0.9'
+  sha256 '0065c58a352e014885ca472fdbe3ac703061d1fb32e4c682c7eb9a019ddf84c0'
+
+  # sourceforge.net is the official download host per the vendor homepage
```
Comment (adityadalal924) [inline Casks/osxfuse-beta.rb]: Delete this comment

**Comment by adityadalal924** [comment]
Please [add an appcast](https://github.com/caskroom/homebrew-cask/issues/16689#issuecomment-170583701)

**Comment by zchee** [comment]
@adidalal Fixed.
PTAL :)

**Comment by adidalal** [comment]
Thank you for the contribution. It was merged directly as 960e1aa25acdb8c32f9251447bb4c9ef7349ef24 to keep commit history cleaner. Your contribution is still credited to you.

File: Casks/osxfuse-beta.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+cask 'osxfuse-beta' do
+  version '3.0.9'
+  sha256 '0065c58a352e014885ca472fdbe3ac703061d1fb32e4c682c7eb9a019ddf84c0'
+
+  url "https://github.com/osxfuse/osxfuse/releases/download/osxfuse-#{version}/osxfuse-#{version}.dmg"
+  appcast 'https://github.com/atom/electron/releases.atom',
```
Comment (tsparber) [inline Casks/osxfuse-beta.rb]: `atom/electron`? -> #1579

**Pull request closed without merging**